internal/config/config.go

```diff
@@ -106,6 +131,16 @@ func AccessTokenPath() (string, error) {
 	return path.Join(dir, "access-token"), nil
 }
 
+// RefreshToken is the path for the refresh token file
+func RefreshTokenPath() (string, error) {
+	dir, err := ConfigDir()
+	if err != nil {
+		return "", err
+	}
+
+	return path.Join(dir, "refresh-token"), nil
```

Comment:
```suggestion
	return filepath.Join(dir, "refresh-token"), nil
```

Or else this won't work on Windows!